src/control/cmd/dmg/auto.go

```diff
@@ -90,7 +90,7 @@ func (cmd *configGenCmd) Execute(_ []string) error {
 	if err := pretty.PrintResponseErrors(resp, &bld); err != nil {
 		return err
 	}
-	cmd.log.Info(bld.String()) // no-op if no host level errors
+	cmd.log.Error(bld.String()) // no-op if no host level errors
```

Comment:
Why error?

Reply:
because it's printing host errors, requested by @mjmac but given I'm going to revise https://github.com/daos-stack/daos/pull/3889 I will move it into there

Reply:
Ah, okay. Sorry, got confused there.